Dockerfile.alpine

```diff
@@ -0,0 +1,42 @@
+FROM python:3.7.3-alpine3.9 AS build
+
+ARG DOCKER_COMPOSE_GITSHA=unknown
+ENV DOCKER_COMPOSE_GITSHA=$DOCKER_COMPOSE_GITSHA
```

Comment:
It's often a good idea to use a different name for the `ARG` than for the `ENV`. Both act in the same "space" (both are set as (environment) variables, which can lead to run situations where the `ENV` is always overridden by the `ARG` (I can find some examples, don't have them at hand)

So, might want to consider, e.g.;

```Dockerfile
ARG GIT_COMMIT=unknown
ENV DOCKER_COMPOSE_GITSHA=$GIT_COMMIT
```

(or vice-versa)

Reply:
Done